src/term/term_to_net.rs

```diff
@@ -144,6 +145,20 @@ fn encode_term(
       inet.link(up, Port(node, 0));
       Ok(Some(Port(node, 0)))
     }
+    Term::Let { pat: LetPat::Tup(l_nam, r_nam), val, nxt } => {
+      let dup = inet.new_node(Dup { lab: u8::try_from(*dups).unwrap() });
```

Comment:
I think it would be cleaner to have a new `NodeKind::Tup` instead of using Dup, even though they have the same behaviour.